internal/datastore/datastore.go

```diff
@@ -168,53 +159,18 @@ func (d *delphisDB) initializeStatements(ctx context.Context) (err error) {
 	}
 
 	// POSTS
-	if d.prepStmts.putPostStmt, err = d.pg.PrepareContext(ctx, `
-		INSERT INTO posts (
-			id,
-			discussion_id,
-			participant_id,
-			post_content_id,
-			quoted_post_id
-		) VALUES ($1, $2, $3, $4, $5)
-		RETURNING
-			id,
-			created_at,
-			updated_at,
-			discussion_id,
-			participant_id,
-			post_content_id,
-			quoted_post_id;`); err != nil {
+	if d.prepStmts.putPostStmt, err = d.pg.PrepareContext(ctx, putPostString); err != nil {
```

Comment:
moved the prepare statements into their own file. this greatly helps with sqlmock as we can just pass in the exact string. will also help keep our tests and queries in sync